src/index.html

```diff
@@ -2,25 +2,18 @@
 <html>
 	<head>
 		<title>Football Info</title>
+		<link rel="stylesheet" href="css/main.min.css"><!---->
```

Comment:
please keep in mind for future: according to conventions on our project we don`t use 'min' suffix for file which are created as a result of gulp tasks